crates/sui-sdk/examples/utils/example_utils.rs

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use reqwest::Client;
+use serde_json::json;
+use shared_crypto::intent::Intent;
+use sui_config::{sui_config_dir, SUI_KEYSTORE_FILENAME};
+use sui_keys::keystore::{AccountKeystore, FileBasedKeystore, InMemKeystore, Keystore};
+
+use sui_types::quorum_driver_types::ExecuteTransactionRequestType;
+
+use sui_sdk::rpc_types::SuiTransactionBlockResponseOptions;
+use sui_sdk::SuiClient;
+use sui_types::base_types::SuiAddress;
+use sui_types::digests::TransactionDigest;
+use sui_types::transaction::Transaction;
+
+// const SUI_FAUCET: &str = "https://faucet.devnet.sui.io/gas"; // devnet faucet
+
+const SUI_FAUCET: &str = "https://faucet.testnet.sui.io/gas"; // testnet faucet
+
+// if you use the sui-test-validator and use the local network; if it does not work, try with port 5003.
+// const SUI_FAUCET: &str = "http://127.0.0.1:9123/gas";
+
+#[allow(dead_code)]
+pub(crate) async fn sui_address_for_examples() -> Result<SuiAddress, anyhow::Error> {
+    let active_address: SuiAddress = SuiAddress::random_for_testing_only();
+    request_tokens_from_faucet(active_address).await?;
+    Ok(active_address)
+}
```

Comment:
Wouldn't recommend doing this for a similar reason to why `InMemKeystore::new_insecure_for_tests` can be confusing -- although these addresses are randomly generated, they could be legitimate addresses, so you are potentially sending coins to random people.

I believe this function is unused though, so best course of action is just to remove it?

Reply:
This is obsolete, yes.